From f264771039f6f96b5b5af1e1451709880f4cd08a Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Mon, 30 Nov 2015 16:50:34 +0100
Subject: [PATCH] xhci: silence TD warning
Patch-Mainline: Never (proper fix would break kABI)
References: bnc#939955

This warning triggers for things that were cleaned up by
the ring expansion code. As we cannot have that in SP3, lowering
the urgency by a notch to not spam logs.

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/host/xhci-ring.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index fcc5f88..a4a141b 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -2453,7 +2453,7 @@ static int handle_tx_event(struct xhci_hcd *xhci,
 		 * TD list.
 		 */
 		if (list_empty(&ep_ring->td_list)) {
-			xhci_warn(xhci, "WARN Event TRB for slot %d ep %d "
+			xhci_dbg(xhci, "WARN Event TRB for slot %d ep %d "
 					"with no TDs queued?\n",
 				  TRB_TO_SLOT_ID(le32_to_cpu(event->flags)),
 				  ep_index);
-- 
2.1.4

